test/GithubApi.Issue.test.js

```diff
@@ -0,0 +1,73 @@
+const agent = require('superagent');
+const { expect } = require('chai');
+
+const urlApi = 'https://api.github.com';
+const namePublicRepo = 'workshop-api-testing-js';
+
+describe('Given a github user', () => {
+  let user;
+  let repository;
+
+  before(async () => {
+    const response = await agent.get(`${urlApi}/user`)
+      .auth('token', process.env.ACCESS_TOKEN)
+      .set('User-Agent', 'agent');
+
+    user = response.body;
+    const listResponse = await agent.get(user.repos_url)
+      .auth('token', process.env.ACCESS_TOKEN)
+      .set('User-Agent', 'agent');
+
+    repository = listResponse.body.find((list) => list.name === namePublicRepo);
+  });
+
+  describe('when wanna all repositories', () => {
```

Comment:
It's not usual to have a when,and, or but without actions afterwards. It's possible the test is disorganized.